docs/docs/dart_api/manager.md

```diff
@@ -1,17 +1,11 @@
 ---
 
-title: Manager
+title: Queries
```

Comment:
IMO we should leave this as-is before the restructuring is more complete. At the moment we'd then effectively have multiple pages on queries which is more confusing than before.

We can name it "Queries with Manager" or something and also point out in the introduction/getting started guide that there are two ways to write these queries (like we already do on the new table page).